spec/units/derivatives_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'hydra/works'
```

Comment:
If you can remove it, remove it.  You might have to add the require statement in the test(s) that do rely on Hydra::Works classes.